ftw/contacts/profiles/default/types/ftw.contacts.ContactFolder.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<object name="ftw.contacts.ContactFolder" meta_type="Dexterity FTI"
+        i18n:domain="ftw.contacts" xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+  <!-- Basic metadata -->
+  <property name="title" i18n:translate="">ContactFolder</property>
+  <property name="description" i18n:translate=""></property>
+  <property name="icon_expr"></property>
+  <property name="allow_discussion">False</property>
+  <property name="global_allow">True</property>
+  <property name="filter_content_types">True</property>
+  <property name="allowed_content_types">
+      <element value="ftw.contacts.Contact" />
+  </property>
+
+  <!-- schema interface -->
+  <property name="schema">ftw.contacts.contents.contactfolder.IContactFolder</property>
+
+  <!-- enabled behaviors -->
+  <property name="behaviors">
+      <element value="plone.app.dexterity.behaviors.metadata.IDublinCore"/>
```

Comment:
Actually we could probably also remove the `IBasic` behavior, as - as far as I remember - the `INameFromTitle` is providing a title-field too.
And we don't really use the description, do we?
Can you test that and change it for the other types too when it works?